docs/docs/reference/built-in-components/gatsby-slice.md

```diff
@@ -120,26 +135,28 @@ export function MyComponent({ sliceName }) {
 // ⚠️ Doesn't work
 
 export function MyComponent() {
-  // Props can't come from function calls
+  // Aliases can't come from function calls
   const aliasName = getAliasFromSomewhere()
   return <Slice alias={aliasName} />
 }
 ```
 
 ```js
-// OK
+// ⚠️ Doesn't work
```

Comment:
😉 https://github.com/gatsbyjs/gatsby/pull/36803

Reply:
lol whoops, thank you